mmv1/third_party/terraform/resources/resource_compute_instance_template.go.erb

```diff
@@ -677,6 +677,15 @@ func resourceComputeInstanceTemplate() *schema.Resource {
 							ForceNew:    true,
 							Description: `The number of threads per physical core. To disable simultaneous multithreading (SMT) set this to 1. If unset, the maximum number of threads supported per core by the underlying processor is assumed.`,
 						},
+<% unless version == 'ga' -%>
+						"visible_core_count": {
+							Type:        schema.TypeInt,
+							Optional:    true,
+							Computed:    false,
```

Comment:
You can remove Computed: false as it is the inferred value

Reply:
Done